app/models/manageiq/providers/azure/cloud_manager/scanning/job.rb

```diff
@@ -100,8 +104,8 @@ def process_cancel(*args)
   def process_abort(*args)
     begin
       delete_snapshot_and_reset_snapshot_mor("aborting")
-      super
```

Comment:
This was calling `super` here and also down on like 112 at the end of the method, so it was failing saying can't transit state finished because it was already in state finished